### PR TITLE
Imu aug fix

### PR DESCRIPTION
**Pull request by rsoussan**
Fixed IMU Augmentor to incrementally add IMU data to latest extrapolated state rather than adding all IMU data since latest graph state message in order to reduce computation time.  Combined with previous PR (see https://github.com/nasa/astrobee/pull/304/files) to include fixes for repeat IMU messages and standstill behavior.

### CODE REVIEW

**Comment by marinagmoreira** [comment]
I'll test this in the lab tomorrow :)

File: localization/imu_augmentor/CMakeLists.txt [hunk 1]
```diff
@@ -38,7 +38,7 @@ find_package(catkin2 REQUIRED COMPONENTS
 
 catkin_package(
   INCLUDE_DIRS include ${GTSAM_INCLUDE_DIR} ${GLOG_INCLUDE_DIRS} 
-  LIBRARIES ${PROJECT_NAME} ${GLOG_LIBRARIES} ${GTSAM_LIBRARIES} 
+  LIBRARIES ${PROJECT_NAME} ${GLOG_LIBRARIES} ${GTSAM_LIBRARIES} gtsam
```
Comment (bcoltin) [inline localization/imu_augmentor/CMakeLists.txt:41]: This should be part of gtsam libraries?

**Review by bcoltin — APPROVED** [review]
Looks good once it's tested

File: localization/imu_augmentor/CMakeLists.txt (hunk 1 above)
Comment (rsoussan) [reply localization/imu_augmentor/CMakeLists.txt:41]: For some reason it's not, not sure what the best way to account for that is but for some packages if I don't include it there I get linker errors

Comment (bcoltin) [reply localization/imu_augmentor/CMakeLists.txt:41]: You could make a FindGSTAM in the cmake folder which fixes this. Will leave it to your judgement whether it's worth the effort.

**Review by marinagmoreira — APPROVED** [review]
Tested this in the lab, the rate of the ctl and fam are now between 70-80Hz and the CPU on the llp is now much lower (the highest CPU load doesn't go over 70% during movement)

**Pull request merged**